src/openforms/formio/rendering/default.py

```diff
@@ -228,8 +228,16 @@ class EditGridNode(ContainerMixin, ComponentNode):
     layout_modifier: str = "editgrid"
     display_value: str = ""
 
+    @property
+    def is_layout(self) -> bool:
+        if self.mode == RenderModes.export:
+            return False
+        return True
```

Comment:
I don't know too much about these rendering classes yet, what does it mean for rendering if `is_layout` is `True` vs `False`? Does `True` imply that the children are rendered individually?

Reply:
yeah these nodes are skipped/ignored in some modes because they don't have an intrinsic value - for example in the email summary only the child elements are displayed, but the column/fieldset/editgrid itself aren't. Similarly, in the export code we have:

```py
        for node in data_nodes.get_children():
            if node.is_layout:
                continue
            yield node
```